next-steps.md

```diff
@@ -0,0 +1,117 @@
+
+
+## Step 3: Validate the template
+
+This stage maps to template status: `Template ready`: The template has reached feature complete and has met the acceptance criteria defined here: [Definition of Done](https://github.com/Azure-Samples/azd-template-artifacts/blob/main/docs/development-guidelines/definition-of-done.md)
+This template is labeled ready for publication to the collection. 
+
+Please follow the links in the [Definition of Done](https://github.com/Azure-Samples/azd-template-artifacts/blob/main/docs/development-guidelines/definition-of-done.md) to complete it. 
+
+Teams can manually run the validation process by adding the validation action [microsoft/template-validation-action](https://github.com/microsoft/template-validation-action) to their repository and following the guidelines in the reference, to make sure they pass validation before requesting inclusion in the collection. 
+
+
+## Step4: Get the results of the validation action and act accordingly
+
+Whether you're running the validation workflow manually in your repository, or it has been run automatically by requesting the collection authors to open a pull-request against the collection list to include it (please read more about this process in the [publishing the template](#publishing-the-template) section, below), you can find the [Auto] issue, with the results of the validation scan.
+
+This action will result in two different template statuses:
+
+### Status: template is valid
+
+Check your repository issues for validation confirmation. You will find an issue with the title `[Auto] AI Gallery Standard Validation.`
+
+If the template passes the validation criteria it will be marked as `PASS` or `CONFORMING` and all the items in the checklist will be checked. If the status is `Template valid`, your template will be manually added to the collection, after review from our collection curators.
+
+If you don’t see your template in the gallery in a reasonable time, reach out to the AI Standardization and AI Gallery team. (@Natalia Venditto , @Kristen Womack, @Grace Kulin ) 
+
+If your template is still under active development, read the [Templates failing after publication](#templates-failing-after-publication) section below. 
+
+
+### Status: template is invalid
+
+If the checks are unsuccessful, the template is invalid and cannot be published. It will be marked as `INVALID` or `NON-CONFORMING`.
+
+There are different levels of validity, and some checks are still under development. 
+
+Please check the [table below](#levels-of-non-conformity-and-expected-action) to understand the levels of non-conformity. 
+
+All items with errors and warnings will display error/warning information. Please follow the “How to fix” links for actionable steps to fix the issues and validate the template for publication. 
+
+### Templates failing after publication
+
+The validation pipeline will be run regularly on all templates. 
```

Comment:
How regularly?

Reply:
Good question. We don't have a decision on cadence, for now. It's something we need to define; this is why I left it open. It's likely it will run nightly after a while.